.circleci/config.yml

```diff
@@ -2,13 +2,10 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/elixir:1.6.1-browsers
+      - image: circleci/elixir:1.6.4-browsers
```

Comment:
This doesn't match what's in the project's `.tool-version`. Would you bump that too? Or bump the version separately in another PR?

Reply:
May as well bump it here, not a big bump anyway.